### PR TITLE
refactor(cli): make `Spinner.color` public

### PR DESCRIPTION
**Pull request by timreichen**
- makes `Spinner.color` prop public.

### CODE REVIEW

File: cli/spinner.ts [hunk 1]
```diff
@@ -83,7 +83,13 @@ export class Spinner {
...
     this.#message = options?.message ?? "";
     this.#interval = options?.interval ?? DEFAULT_INTERVAL;
-    this.#color = options?.color ? COLORS[options.color] : undefined;
+    this.color = options?.color;
+  }
+  set color(value: Color | undefined) {
+    this.#color = value ? COLORS[value] : undefined;
+  }
+  get color() {
+    return this.#color;
```
Comment (iuioiua) [inline cli/spinner.ts:92]:
```suggestion
  }
  
  set color(value?: Color) {
    this.#color = value ? COLORS[value] : undefined;
  }
  
  get color() {
    return this.#color;
```
Nits

**Review by iuioiua — CHANGES_REQUESTED** [review]
Can we add a short message saying this can be changed on the fly, like the `message` property?

File: cli/spinner.ts (hunk 1 above)
Comment (timreichen) [reply cli/spinner.ts:98]: A 'set' accessor cannot have an optional parameter.

Comment (iuioiua) [reply cli/spinner.ts:98]: Oh. Makes sense in hindsight...

**Review by iuioiua — APPROVED** [review]
LGTM

**Review by kt3k — APPROVED** [review]
LGTM

**Pull request merged**